markdown-parsing.js

```diff
@@ -180,6 +183,22 @@ const filters: {[key: string]: (HTMLElement) => string | HTMLElement} = {
   },
   UL(el) {
     return el
+  },
+  DIV(el) {
+    if (el.classList.contains('js-suggested-changes-blob')) {
```

Comment:
Could this somehow live in the application code since it seems tightly coupled with it?